opus/application/static_media/js/opus.js

```diff
@@ -321,7 +320,7 @@ var opus = {
         // the application default widgets
 
         clearInterval(opus.main_timer);  // stop polling for UI changes for a moment
-        $("#search_widgets").empty(); // remove all widgets on the screen
+        $("#op-search-widgets").empty(); // remove all widgets on the screen
```

Comment:
Debby and I had a talk about empty() awhile back. My undertstanding is that "empty" detaches the children and their contents from the DOM, but still leaves the elements in the browser, essentially wasting memory. They never go away. Instead we should be using remove(), which both detaches and deletes the elements from memory. Debby do you agree?

Reply:
yes - see http://mundrisoft.com/tech-bytes/jquery-difference-between-remove-empty-and-detach-methods/

Reply:
Thanks for the suggestion, the code is updated to use .remove() instead of .empty().